gc/base/standard/Scavenger.cpp

```diff
@@ -1677,6 +1677,22 @@ MM_Scavenger::copy(MM_EnvironmentStandard *env, MM_ForwardedHeader* forwardedHea
 	return destinationObjectPtr;
 }
 
+MMINLINE void
+MM_Scavenger::fixupForwardedObject(MM_ForwardedHeader* forwardedHeader, omrobjectptr_t destinationObjectPtr, uintptr_t objectAge) {
+	/* Object model specific fixup, like age */
+	_extensions->objectModel.fixupForwardedObject(forwardedHeader, destinationObjectPtr, objectAge);
+
+#if defined(OMR_ENV_DATA64)
+	if(_extensions->objectModel.isIndexable(destinationObjectPtr)) {
+		/* Updates internal data address of indexable objects. Every indexable object have a void *dataAddr
```

Comment:
Since dataAddr is OpenJ9 specific pointer, wording should be somewhat softer: 'indexable object _may_ have'.
We shouldn't even be very specific about its type (void *) here, but we can still generically talk abut dataAddr. Other languages may define it differently, though.

Reply:
That makes sense, I'll reword it

Reply:
I updated it and gave a brief use case which is OpenJ9